diff --git a/modules/custom/country_music/src/Controller/SongController.php b/modules/custom/country_music/src/Controller/SongController.php
index 5f1201b..81b124f 100644
--- a/modules/custom/country_music/src/Controller/SongController.php
+++ b/modules/custom/country_music/src/Controller/SongController.php
@@ -7,11 +7,6 @@
 
 class SongController extends ControllerBase {
   public function writeSong($noun) {
-    if ($noun === null) {
-      $noun = $this->config('country_music.default')
-        ->get('noun');
-    }
-
     $songGenerator = \Drupal::getContainer()
       ->get('country_music.song_generator');
     $title = $songGenerator->generateSong($noun);
diff --git a/modules/custom/country_music/src/Service/SongGenerator.php b/modules/custom/country_music/src/Service/SongGenerator.php
index 45c393d..e884bbc 100644
--- a/modules/custom/country_music/src/Service/SongGenerator.php
+++ b/modules/custom/country_music/src/Service/SongGenerator.php
@@ -5,6 +5,11 @@
 class SongGenerator {
   public function generateSong($noun)
   {
+    if ($noun === null) {
+      $noun = $this->config('country_music.default')
+        ->get('noun');
+    }
+
     $verbs = ['drive', 'drink', 'dance'];
     $adjectives = ['Honky Tonk', 'country', 'back road', 'boot scootin'];
     $templates = [
