diff --git a/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
new file mode 100644
index 0000000..a15c0a1
--- /dev/null
+++ b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
@@ -0,0 +1,7 @@
+<?php
+
+namespace Drupal\country_music\EventSubscriber;
+
+class CountrySingerSubscriber {
+
+}
\ No newline at end of file
