diff --git a/modules/custom/country_music/country_music.services.yml b/modules/custom/country_music/country_music.services.yml
index 66bf20f..f4e5ed7 100644
--- a/modules/custom/country_music/country_music.services.yml
+++ b/modules/custom/country_music/country_music.services.yml
@@ -2,3 +2,7 @@ services:
   country_music.song_generator:
     class: Drupal\country_music\Service\SongGenerator
     arguments: ['@config.factory']
+
+  country_music.country_singer_subscriber:
+    class: Drupal\country_music\EventSubscriber\CountrySingerSubscriber
+    arguments: []
diff --git a/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
index a15c0a1..ad59723 100644
--- a/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
+++ b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
@@ -2,6 +2,17 @@
 
 namespace Drupal\country_music\EventSubscriber;
 
-class CountrySingerSubscriber {
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 
+class CountrySingerSubscriber implements EventSubscriberInterface {
+  public function onKernelRequest(GetResponseEvent $event) {
+    drupal_set_message('Welcome to Music City USA');
+  }
+
+  public static function getSubscribedEvents() {
+    return [
+      'kernel.request' => 'onKernelRequest'
+    ];
+  }
 }
\ No newline at end of file
