diff --git a/modules/custom/country_music/country_music.services.yml b/modules/custom/country_music/country_music.services.yml
index c183445..4350392 100644
--- a/modules/custom/country_music/country_music.services.yml
+++ b/modules/custom/country_music/country_music.services.yml
@@ -5,6 +5,6 @@ services:
 
   country_music.country_singer_subscriber:
     class: Drupal\country_music\EventSubscriber\CountrySingerSubscriber
-    arguments: []
+    arguments: ['@messenger']
     tags:
       - { name: event_subscriber }
diff --git a/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
index ad59723..85eb60f 100644
--- a/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
+++ b/modules/custom/country_music/src/EventSubscriber/CountrySingerSubscriber.php
@@ -2,12 +2,20 @@
 
 namespace Drupal\country_music\EventSubscriber;
 
+use Drupal\Core\Messenger\MessengerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 
 class CountrySingerSubscriber implements EventSubscriberInterface {
+
+  private $messenger;
+
+  public function __construct(MessengerInterface $messenger) {
+    $this->messenger = $messenger;
+  }
+
   public function onKernelRequest(GetResponseEvent $event) {
-    drupal_set_message('Welcome to Music City USA');
+    $this->messenger->addStatus('Welcome to Music City USA');
   }
 
   public static function getSubscribedEvents() {
