diff --git a/modules/custom/country_music/country_music.services.yml b/modules/custom/country_music/country_music.services.yml
index 4350392..94e5054 100644
--- a/modules/custom/country_music/country_music.services.yml
+++ b/modules/custom/country_music/country_music.services.yml
@@ -8,3 +8,7 @@ services:
     arguments: ['@messenger']
     tags:
       - { name: event_subscriber }
+  country_music.route_subscriber:
+    class: Drupal\country_music\Routing\RouteSubscriber
+    tags:
+      - { name: event_subscriber }
diff --git a/modules/custom/country_music/src/Routing/RouteSubscriber.php b/modules/custom/country_music/src/Routing/RouteSubscriber.php
new file mode 100644
index 0000000..6efada3
--- /dev/null
+++ b/modules/custom/country_music/src/Routing/RouteSubscriber.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\country_music\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Class RouteSubscriber.
+ *
+ * Listens to the dynamic route events.
+ */
+class RouteSubscriber extends RouteSubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+  }
+}
