diff --git a/modules/custom/country_music/src/Plugin/Block/SongBlock.php b/modules/custom/country_music/src/Plugin/Block/SongBlock.php
new file mode 100644
index 0000000..3bdbf0f
--- /dev/null
+++ b/modules/custom/country_music/src/Plugin/Block/SongBlock.php
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\country_music\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a 'SongBlock' block.
+ *
+ * @Block(
+ *  id = "song_block",
+ *  admin_label = @Translation("Song block"),
+ * )
+ */
+class SongBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [];
+    $build['song_block']['#markup'] = 'Implement SongBlock.';
+
+    return $build;
+  }
+
+}
