diff --git a/modules/custom/country_music/src/Controller/SongController.php b/modules/custom/country_music/src/Controller/SongController.php
index df31bf1..3fab8f8 100644
--- a/modules/custom/country_music/src/Controller/SongController.php
+++ b/modules/custom/country_music/src/Controller/SongController.php
@@ -2,10 +2,13 @@
 
 namespace Drupal\country_music\Controller;
 
-use Symfony\Component\HttpFoundation\Response;
-
 class SongController {
   public function writeSong($noun) {
-    return new Response(sprintf('I rode my %s, through some mud.', $noun));
+    $title = sprintf('I rode my %s, through some mud.', $noun);
+
+    return [
+      '#type' => 'markup',
+      '#markup' => $title,
+    ];
   }
 }
