diff --git a/modules/custom/country_music/country_music.routing.yml b/modules/custom/country_music/country_music.routing.yml
index 54ef36b..8f5f927 100644
--- a/modules/custom/country_music/country_music.routing.yml
+++ b/modules/custom/country_music/country_music.routing.yml
@@ -3,7 +3,7 @@ country_music.song.list:
   defaults:
     _controller: 'Drupal\country_music\Controller\SongController::writeSong'
     _title: 'Write the next hit!'
-    noun: truck
+    noun: null
   requirements:
     _permission: 'access content'
 
diff --git a/modules/custom/country_music/src/Controller/SongController.php b/modules/custom/country_music/src/Controller/SongController.php
index 3fab8f8..6298bc6 100644
--- a/modules/custom/country_music/src/Controller/SongController.php
+++ b/modules/custom/country_music/src/Controller/SongController.php
@@ -2,8 +2,15 @@
 
 namespace Drupal\country_music\Controller;
 
-class SongController {
+use Drupal\Core\Controller\ControllerBase;
+
+class SongController extends ControllerBase {
   public function writeSong($noun) {
+    if ($noun === null) {
+      $noun = $this->config('country_music.default')
+        ->get('noun');
+    }
+
     $title = sprintf('I rode my %s, through some mud.', $noun);
 
     return [
