diff --git a/modules/custom/country_music/country_music.services.yml b/modules/custom/country_music/country_music.services.yml
new file mode 100644
index 0000000..08c0532
--- /dev/null
+++ b/modules/custom/country_music/country_music.services.yml
@@ -0,0 +1,4 @@
+services:
+  country_music.song_generator:
+    class: Drupal\country_music\Service\SongGenerator
+    arguments: []
diff --git a/modules/custom/country_music/src/Controller/SongController.php b/modules/custom/country_music/src/Controller/SongController.php
index 9113de9..5f1201b 100644
--- a/modules/custom/country_music/src/Controller/SongController.php
+++ b/modules/custom/country_music/src/Controller/SongController.php
@@ -12,7 +12,8 @@ public function writeSong($noun) {
         ->get('noun');
     }
 
-    $songGenerator = new SongGenerator();
+    $songGenerator = \Drupal::getContainer()
+      ->get('country_music.song_generator');
     $title = $songGenerator->generateSong($noun);
 
     return [
