diff --git a/modules/custom/country_music/config/install/country_music.default.yml b/modules/custom/country_music/config/install/country_music.default.yml
new file mode 100644
index 0000000..4aa6c74
--- /dev/null
+++ b/modules/custom/country_music/config/install/country_music.default.yml
@@ -0,0 +1 @@
+country_music:
diff --git a/modules/custom/country_music/country_music.links.menu.yml b/modules/custom/country_music/country_music.links.menu.yml
new file mode 100644
index 0000000..255379c
--- /dev/null
+++ b/modules/custom/country_music/country_music.links.menu.yml
@@ -0,0 +1,7 @@
+country_music.default_form:
+  title: 'Music Form'
+  route_name: country_music.default_form
+  description: 'Help make great music'
+  parent: system.admin_config_system
+  weight: 99
+
diff --git a/modules/custom/country_music/country_music.routing.yml b/modules/custom/country_music/country_music.routing.yml
index cd87ce5..bcd6139 100644
--- a/modules/custom/country_music/country_music.routing.yml
+++ b/modules/custom/country_music/country_music.routing.yml
@@ -5,3 +5,14 @@ country_music.song.list:
     noun: truck
   requirements:
     _permission: 'access content'
+
+country_music.default_form:
+  path: '/admin/config/country_music/default'
+  defaults:
+    _form: '\Drupal\country_music\Form\DefaultForm'
+    _title: 'DefaultForm'
+  requirements:
+    _permission: 'access administration pages'
+  options:
+    _admin_route: TRUE
+
diff --git a/modules/custom/country_music/src/Form/DefaultForm.php b/modules/custom/country_music/src/Form/DefaultForm.php
new file mode 100644
index 0000000..240c0b4
--- /dev/null
+++ b/modules/custom/country_music/src/Form/DefaultForm.php
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\country_music\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class DefaultForm.
+ */
+class DefaultForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'country_music.default',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'default_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('country_music.default');
+    $form['noun'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Noun'),
+      '#maxlength' => 64,
+      '#size' => 64,
+      '#default_value' => $config->get('noun'),
+    ];
+    return parent::buildForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+    parent::validateForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    parent::submitForm($form, $form_state);
+
+    $this->config('country_music.default')
+      ->set('noun', $form_state->getValue('noun'))
+      ->save();
+  }
+
+}
